api-reference/10 UI Widgets/dxSortable/3 Methods/update().md

```diff
@@ -3,7 +3,18 @@ id: dxSortable.update()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Updates **Sortable**'s dimensions. Call this method after items are added or their dimensions are changed during dragging.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Depending on the framework, use this method in different places:
```

Comment:
I don't think "after a markup update" is a place. It sounds more like a point in time.
Can we make this statement more generic? For example:

`Depending on the framework, use this method as follows:`

Reply:
Yes, it is better.

Reply:
Thank you.